### ISSUE TITLE
More detailed instructions

### ISSUE THREAD

**Issue by NotepadPlusUser** [darktrojan/openwith#246]
Thank you very much for this extremely useful addon. that I have been using for some time 

I had to reinstall Python, and I totally lost the plot with the Python script business.  Could I suggest that some more detailed and supportive instructions are given for this procedure.

* First, where should the file be downloaded to?  I put it into a Vivaldi configuration directory that I had created, then when the add-on didn't work (see the next point) I copied it to the Python installation directory "C:\Program Files (x86)\Python37-32\" as well.

* Secondly, the add-on still didn't work.  
 --- I read the instructions again and realised that I was using the RunBox instead of the DOSBox command prompt, so I launched this and ran it from there.
 --- It still didn't work, so I ran it from an Administrator DOSbox command prompt.
 --- Now it works.  And then I remembered that I had gone through this same frustrating set of procedures some time ago when I had first installed 'Open With'. 

CONCLUSION:  Explaining where to put the file, and the fact that it needs to be run from an Administrator DOSBox, both need to be in the instructions.  Most of my friends would have given up well before this.  Also, how to launch an Administrator DOSBox needs to be explained --- most users will never have heard of it.

But now it works like a charm, and I use it constantly (Vivaldi, Firefox, IE, Edge).  Thanks again.

**Comment by computer-guru** [comment]
I agree with this post. I ran into issues before I could get it to work. Some issues with registry and folder permissions, because the script was not ran as administrator.
